torch/nn/functional.py

```diff
@@ -17,6 +17,11 @@
 
 _VF = torch._C._VariableFunctions
 
+def weak_script(fn):
```

Comment:
This redefinition is not great, but there's a circular dependency between `torch.jit` and `torch.nn` if it's included here. Any ideas to fix?